js/script.js

```diff
@@ -0,0 +1,36 @@
+/** @format */
+
+// const logo = document.querySelector('.logo');
+const navMenu = document.querySelector('.nav-menu');
+const hamburger = document.querySelector('.hamburger');
+const navItem = document.querySelectorAll('.nav-item');
+
+const openMenu = () => {
+  // logo.classList.toggle('active');
+  navMenu.classList.remove('desktop');
+  navMenu.classList.toggle('active');
+  hamburger.classList.toggle('active');
+};
+
+const closeMenu = () => {
+  navMenu.classList.add('desktop');
+  // logo.classList.remove('active');
```

Comment:
Please remove unused pieces of code _(here, line 3 and 9)_ from your code base as this will make for cleaner and more readable code 👍